canvas_modules/common-canvas/src/common-properties/components/table-buttons/table-buttons.scss

```diff
@@ -1,49 +0,0 @@
-/*
- * Copyright 2022 Elyra Authors
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-.properties-custom-table-buttons {
-	.toolbar-div {
-		height: $spacing-07;
-
-		.toolbar-left-bar {
-			padding-right: 0;
-
-			.toolbar-item button {
-				height: $spacing-07;
-			}
-
-			.toolbar-item-content {
-				padding: $spacing-02 $spacing-03;
-				height: $spacing-07;
-				min-width: $spacing-07;
-				align-items: center;
-			}
-
-			.toolbar-overflow-item button {
-				height: $spacing-07;
-			}
-
-			.toolbar-popover-list {
-				top: $spacing-07;
-				right: 0;
```

Comment:
Hey Nabil, we will still need this style for the overflow icon inside tables, so the menu will appear to the left instead of the right
![image](https://user-images.githubusercontent.com/1628100/199135803-d5d1d64d-e85d-4a8f-8521-d57b152fbe11.png)

Reply:
<img width="376" alt="image" src="https://user-images.githubusercontent.com/89609176/199341143-245ba8f5-b7b5-484e-9b0f-b62fe3f1da69.png">